rti/mosaic-rti-api/src/main/java/org/eclipse/mosaic/rti/UNITS.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.rti;
+
+public final class UNITS {
+
+    private UNITS() {
+        // access to constants only
+    }
+
+    /**
+     * Distance of one meter.
+     */
+    public static final double METER = 1;
```

Comment:
Is meter the smallest resolution we use for distances? Coming from TIME, we start at a much smaller scale

Reply:
With time we use discrete values, therefore we use nanoseconds as the smallest possible value. For distance and speed we use real numbers and per (unwritten) convention we use "meter" and "m/s" as this is the most commonly used unit.